### PR TITLE
style/small ui issues

### PR DESCRIPTION
**Pull request by Boyuan-Deng**
## Description

closes: https://www.notion.so/mantanetwork/2becffd42f3a4298a9057b0cc6666746?v=6d1a0f73df2d45f1877bda3c5d69c811&p=fa997097c2e44016ba4d9053fee8d869&pm=s

A couple things different than design on purpose
- text size & logo size for asset
- move the select drop down lower to show balance
 
image
image

---

Before we can merge this PR, please make sure that all the following items have been checked off. If any of the checklist items are not applicable, please leave them but write a little note why.

- [ ] Linked to Github issue with discussion and accepted design OR have an explanation in the PR that describes this work
- [ ] Re-reviewed files changed in the Github PR explorer

## If PR to `staging`
- [ ] Updated relevant documentation in the code
- [ ] Test code changes manually, and describe your procedure in a comment on this PR
- [ ] Mark this PR with the correct milestone

## If PR to `main`
- [ ] Update the version number in `package.json` and `src/config/common.json`
- [ ] Update the minimum required signer version number in `src/config/common.json` if applicable
- [ ] Test following the procedure in `docs/manual_test.md`
- [ ] Update `CHANGELOG.md`
- [ ] Make sure that all PR's to `manta-signer` and `sdk` on which this PR depends are merged

### CODE REVIEW

**Comment by Kevingislason** [comment]
-The corners in the design for the dropdowns should be rounded to match the design
-This PR shrinks the chain selector icon, but the text also should be shrunk by quite a bit to match the design
-"The line color is different from UI design" -> This is still unresolved

**Comment by zhangjing28** [comment]
> * move the select drop down lower to show balance

Why did that on purpose? It will confuse the user whether the Dropdown List is related to the Dropdown component. If the balance is important to users to choose which token, maybe it's better we display the the balance of each token in the Dropdown list.

**Comment by zhangjing28** [comment]
The Dropdown hover design should be like this. Our design already added this to this [notion](https://www.notion.so/mantanetwork/Some-UI-issues-fa997097c2e44016ba4d9053fee8d869)
image

**Comment by Boyuan-Deng** [comment]
> -The corners in the design for the dropdowns should be rounded to match the design -This PR shrinks the chain selector icon, but the text also should be shrunk by quite a bit to match the design -"The line color is different from UI design" -> This is still unresolved

In the new implementation, I tried to match everything with the new design
https://www.notion.so/mantanetwork/2becffd42f3a4298a9057b0cc6666746?v=6d1a0f73df2d45f1877bda3c5d69c811&p=fa997097c2e44016ba4d9053fee8d869&pm=s

File: tailwind.config.js [hunk 1]
```diff
@@ -75,11 +81,15 @@ module.exports = {
...
         primary: 'var(--color-border-primary)',
         secondary: 'var(--color-bg-button-primary)',
-        warning: '#FF8A00'
-      }
+        warning: '#FF8A00',
+        '#FFFFFF1A': '#FFFFFF1A',
+        '#2B49EA': '#2B49EA'
+      },
+      divideColor: {
+        '#FFFFFF1A': '#FFFFFF1A'
```
Comment (Kevingislason) [inline tailwind.config.js:91]: It would be better to give these colors human-meaningful names

**Review by Kevingislason — COMMENTED** [review]
Good stuff, some of these changes improve the UI quite noticeably.

A few things:

- [x] The chain selector button seems too small, both the text and the icon. Based on the figma, text should be the same font size as the connect wallet button (though not bold). The icons appear to be larger than the icons within the send / bridge form.

- [x] The chain selector icon shrinks if I shrink the browser window's width (only the Dolphin icon)

- [ ] When you hover dropdown items, the highlighted portion should have its corners rounded if it's the first and/or last item in the list. On the bridge page, corners are always rounded, and on the zkTransact page they are never rounded.

- [ ] The chain selector uses a special font, which I think we are missing. In general, the front end uses several different fonts in the design, but currently we are only using one. If updating fonts isn't something you can do pretty easily, feel free to save it for another PR.

- [x] The text on the chain select component should be `Select a network`

- [x] The connect wallet modal in the design is a bit less wide

- [ ] The initial setup modal ("Start to zkTransact") has a light blue outline around the browser wallet options / the connected wallets on the final screen [follow up with Chloe and Yehan next week]

- [ ] The light blue border around the public/private toggle buttons is relatively less thick in the design

**Comment by Boyuan-Deng** [comment]
> Good stuff, some of these changes improve the UI quite noticeably.
> 
> A few things:
> 
> * [x]  The chain selector button seems too small, both the text and the icon. Based on the figma, text should be the same font size as the connect wallet button (though not bold). The icons appear to be larger than the icons within the send / bridge form.
> * [x]  The chain selector icon shrinks if I shrink the browser window's width (only the Dolphin icon)
> * [x]  When you hover dropdown items, the highlighted portion should have its corners rounded if it's the first and/or last item in the list. On the bridge page, corners are always rounded, and on the zkTransact page they are never rounded.
> * [x]  The chain selector uses a special font, which I think we are missing. In general, the front end uses several different fonts in the design, but currently we are only using one. If updating fonts isn't something you can do pretty easily, feel free to save it for another PR.
> * [x]  The text on the chain select component should be `Select a network`
> * [x]  The connect wallet modal in the design is a bit less wide
> * [x]  The initial setup modal ("Start to zkTransact") has a light blue outline around the browser wallet options / the connected wallets on the final screen [follow up with Chloe and Yehan next week]
> * [x]  The light blue border around the public/private toggle buttons is relatively less thick in the design

The chain icon is actually the same size as the assetSelect icon according to the design under inspect
I only updated the fonts for chainSelector but there are other fonts get updated which I purposefully ignore since there are conflicts with old design.

**Comment by Kevingislason** [comment]
> The chain icon is actually the same size as the assetSelect icon according to the design under inspect I only updated the fonts for chainSelector & connect walllet & connect signer

The network icons when you open the chain select menu are the same size as the assetSelect icon (24 x 24) in the design, but the network icon you click to open the chain select menu is 28x28 in the design. lmk if you're not seeing this.

File: tailwind.config.js [hunk 2]
```diff
@@ -30,7 +30,12 @@ module.exports = {
         '128':'32rem',
         '108':'27rem',
         '113.5':'28.375rem',
-        '21':'5.375rem'
+        '21':'5.375rem',
```
Comment (Kevingislason) [inline tailwind.config.js:33]: 21 would correspond to 5.25 rem

**Review by Kevingislason — COMMENTED** [review]
- [ ]  I think this issue still applies: "The initial setup modal ("Start to zkTransact") has a light blue outline around the browser wallet options / the connected wallets on the final screen [follow up with Chloe and Yehan next week"
- [ ] I also think this still applies; "The connect wallet modal in the design is a bit less wide"
- [ ] If you zoom in on the dropdowns, there's some weird additional transparent space
Screen Shot 2023-01-25 at 7 56 33 PM

**Review by Kevingislason — COMMENTED** [review]
Okay, one last thing--the change you made to reduce the width of the initial setup modal also shrunk the initial setup modal, which I think we actually don't want to change

**Review by Kevingislason — APPROVED** [review]

**Review by zhangjing28 — APPROVED** [review]

**Pull request merged**